java-maps-places/.repo-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "api_shortname": "maps-places",
+  "name_pretty": "Places API (New)",
```

Comment:
Is this the actual requested pretty name, with the `(New)`?

Reply:
According to their service yaml file, yes - but I'll confirm on the bug.

Reply:
Confirmed that is indeed the name and matches the name in [Pantheon](https://pantheon.corp.google.com/marketplace/product/google/places.googleapis.com?project=boq-prod-mapsplaces)